packages/build/src/plugins/resolve.js

```diff
@@ -186,38 +184,29 @@ const handleIntegrations = async function ({
     buildDir,
     pluginsEnv,
   })
-
-  if (toInstall.length === 0) {
-    return []
-  }
-
   return Promise.all(
-    toInstall.map((integration) =>
-      resolveIntegration({
+    integrationsWithBuildPlugins.map(async (integration) => {
+      // TODO(ndhoule): When developing locally, the user can accidentally set an extension name in
```

Comment:
Though I'm fairly sure we should do this, I'm going to punt on this TODO for now and let it be a point of feedback from users. It'll take a little while, and I think shipping native-dependency support in SDK-injected functions is higher priority than polishing this rough UX edge.